.github/workflows/docker-build.yaml

```diff
@@ -570,12 +570,22 @@ jobs:
                     name: docker-compose-cypress
             -   name: Stop previous running application
                 run: docker compose down
+            -   name: Prepare docker-compose.yml file from docker-compose.github-actions.cypress.yml.dist
+                run: |
+                    sh .github/prepare-docker-compose-file.sh \
+                    "./docker/conf/docker-compose.github-actions.cypress.yml.dist" \
+                    "./docker-compose.yml" \
+                    "${{ needs.variables.outputs.DOCKER_PHP_FPM_BASE_REPOSITORY_TAG }}" \
+                    "${{ needs.variables.outputs.DOCKER_STOREFRONT_REPOSITORY_TAG }}" \
+                    "${{ needs.variables.outputs.DOCKER_ELASTICSEARCH_REPOSITORY_TAG }}"
```

Comment:
I see one problem with this - before docker-composes were in artifacts, so in case of need to debug something, you just downloaded them and used them.

Also you did not removed all docker composes from artifacts

Reply:
docker composes in artifacts are still used in the default build.
The problem with the fork build is, that we cannot use the docker-compose from artifacts because the step with the build image is not run, therefore the image in the docker-compose in the artifacts does not exist.